labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -58,9 +54,7 @@ object ListManipulationExercise01 {
    *  - custom made
    *  - ... etc 
    */
-  def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
-  }
+  def concatLists[T](l1: List[T], l2: List[T]): List[T] = l1:::l2
```

Comment:
Interestingly, `++` does the same as `:::`, but it works for every kind of collection, including chaining iterators together. I believe `:::` is a legacy syntax.